src/ir/context.rs

```diff
@@ -731,6 +745,7 @@ impl<'ctx> BindgenContext<'ctx> {
     /// Iterate over all items and replace any item that has been named in a
     /// `replaces="SomeType"` annotation with the replacement type.
     fn process_replacements(&mut self) {
+        let _t = self.timer("process_replacements");
```

Comment:
Yep, I see you figured out how to do RAII variables that are only used for their `Drop` without getting unused variable warnings :)

Reply:
It's a learning process :)